comps/image2video/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2024 Intel Corporation
```

Comment:
Please add test for this new microservice.

Reply:
Added test.